core/src/main/scala/cell/HandlerPool.scala

```diff
@@ -37,23 +39,23 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
    *
    * @param key The key to resolve this cell if in a cycle or no result computed.
    * @param init A callback to return the initial value for this cell and to set up dependencies.
-   * @param lattice The lattice of which the values of this cell are taken from.
+   * @param updater The lattice of which the values of this cell are taken from.
    * @return Returns a cell.
    */
-  def mkCell[K <: Key[V], V](key: K, init: (Cell[K, V]) => Outcome[V])(implicit lattice: Lattice[V]): Cell[K, V] = {
-    CellCompleter(key, init)(lattice, this).cell
+  def mkCell[K <: Key[V], V](key: K, init: (Cell[K, V]) => Outcome[V])(implicit updater: Updater[V]): Cell[K, V] = {
+    CellCompleter(key, init)(updater, this).cell
   }
 
   /**
    * Returns a new cell in this HandlerPool.
    *
    * Creates a new, completed cell with value `v`.
    *
-   * @param lattice The lattice from which the values of this cell are taken
+   * @param updater The lattice from which the values of this cell are taken
```

Comment:
"The lattice from which the values of this cell are taken" -> "The updater used to update the value of this cell."